package.json

```diff
@@ -65,7 +65,7 @@
     "dev": "NODE_ENV=development nodemon src/server.js --exec babel-node --inspect=9229",
     "build": "NODE_ENV=production gulp",
     "build:doc": "bootprint openapi doc/api.yaml public/doc/api",
-    "test": "NODE_ENV=development jest __tests__",
+    "test": "NODE_ENV=development jest --runInBand __tests__",
```

Comment:
Why do we need `--runInBand`? Was it for debugging?

Reply:
Yes, it prints out the console logs. Will remove it.